adafruit_esp32spi/adafruit_esp32spi_wifimanager.py

```diff
@@ -203,7 +203,7 @@ def connect_enterprise(self):
                 failure_count = 0
                 self.pixel_status((0, 100, 0))
                 sleep(1)
-            except (ValueError, RuntimeError) as error:
+            except (ValueError, OSError) as error:
```

Comment:
A `ValueError` now indicates something that is never going to work, so I think it should no longer be caught here, and instead should just propagate up.
```suggestion
            except OSError as error:
```